mmtk/src/active_plan.rs

```diff
@@ -14,13 +14,13 @@ impl ActivePlan<OpenJDK> for VMActivePlan {
         &*SINGLETON.plan
     }
 
-    unsafe fn is_mutator(tls: OpaquePointer) -> bool {
-        ((*UPCALLS).is_mutator)(tls)
+    fn is_mutator(tls: VMThread) -> bool {
+        unsafe { ((*UPCALLS).is_mutator)(tls) }
     }
 
-    unsafe fn mutator(tls: OpaquePointer) -> &'static mut Mutator<OpenJDK> {
-        let m = ((*UPCALLS).get_mmtk_mutator)(tls);
-        &mut *m
+    fn mutator(tls: VMMutatorThread) -> &'static mut Mutator<OpenJDK> {
+        let m = unsafe { ((*UPCALLS).get_mmtk_mutator)(tls) };
```

Comment:
Probably wrap the two lines in one `unsafe` block instead of two?

Reply:
Sure.